.github/wiki/Getting-Started.md

```diff
@@ -15,38 +16,50 @@ Onefetch is installed, then what?
 ### Misc
 
 By [**@spenserblack**](https://github.com/spenserblack)
+
 ```sh
 # Runs `onefetch -a Assembly`, `onefetch -a C`, etc.
 onefetch -l | tr "[:upper:] " "[:lower:]-" | while read line; do echo "$line"; onefetch -a $line; done;
 ```
-By [**@quazar-omega**](https://github.com/quazar-omega)
+
+### Automatic repo detection and running
+
+If you want to automate the detection and running of `onefetch` every time you `cd` into a repository you can leverage one of the methods below:
+
+#### 1. Bash / Zsh
+
+> By [**@quazar-omega**](https://github.com/quazar-omega)
```

Comment:
If it's not too much trouble, can you undo these quotes (`> `)? I guess they were added for emphasis. My concern is that they get rendered to `<blockquote>` in HTML, which can convey unintended semantic meaning and be harmful for screen readers.

Reply:
sure. good point